secp-api/src/main/java/org/bitcoinj/secp/api/internal/ByteUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023-2024 secp256k1-jdk Developers.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bitcoinj.secp.api.internal;
+
+import java.util.Comparator;
+
+/**
+ *
+ */
+public class ByteUtils {
+    /**
+     * Provides a byte array comparator.
+     * @return A comparator for byte[]
+     */
+    public static Comparator<byte[]> arrayUnsignedComparator() {
+        return ARRAY_UNSIGNED_COMPARATOR;
+    }
+
+    // In Java 9, this can be replaced with Arrays.compareUnsigned()
```

Comment:
I assume you're planning for JDK 8? Otherweise you as per comment you could replace this.

Reply:
Yes, planning for JDK 8!